### ISSUE TITLE
need a set of test cases.

### ISSUE THREAD

**Issue by debghs** [Alitindrawan24/Binary-Calculator#54]
I have noticed that with some of the PRs merged, the logic of the calculator is getting broken. 
Even though I have fixed all the bugs in my last commit (PR #53), this might happen again later on.

I suggest that @Alitindrawan24, please implement a process such that, before merging any new PR, the commit passes through a set of test cases that contain all the different types of cases where the logic might get broken, i.e. edge cases.

I may work on this if you wish, but I think you(@Alitindrawan24 ) should implement this asap(before any further PRs are merged).

**Comment by Alitindrawan24** [comment]
Yes I think we need a lot of test case in this project to make sure any logic is still work properly. Can you do this to start the biolerplate of the test case ? Thank you

**Comment by debghs** [comment]
Yeah, sure.
